estate/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import estate_properties
```

Comment:
We usually name a model and the file in the singular form
```suggestion
from . import estate_property
```